src/palimpzest/operators/operators.py

```diff
@@ -167,38 +165,29 @@ def _createPhysicalPlans(self, max: int=10, shouldProfile: bool=False) -> Physic
 
         return physicalPlans
 
-    def createPhysicalPlanCandidates(self, sampler: Sampler=None, shouldProfile: bool=False) -> List[PhysicalPlan]:
+    def createPhysicalPlanCandidates(self, cost_estimate_sample_data: List[Dict[str, Any]]=None, shouldProfile: bool=False) -> List[PhysicalPlan]:
```

Comment:
It's a drag to have this stuff in just a Dict....  should this be a proper class, or is the flexibility of this needed?